pytransform3d/plot_utils/_artists.py

```diff
@@ -299,3 +299,108 @@ def draw(self, renderer):
     def do_3d_projection(self, renderer=None):
         # This supports both matplotlib 3.4 and 3.5
         return 0
+
+
+class Camera(artist.Artist):
+    """A Matplotlib artist that displays a camera.
+
+    This function is inspired by Blender's camera visualization. It will
+    show the camera center, a virtual image plane, and the top of the virtual
+    image plane.
+
+    Parameters
+    ----------
+    M : array-like, shape (3, 3)
+        Intrinsic camera matrix that contains the focal lengths on the diagonal
+        and the center of the the image in the last column. It does not matter
+        whether values are given in meters or pixels as long as the unit is the
+        same as for the sensor size.
+
```

Comment:
A2B is missing in the docstring

Reply:
Thanks! I added docstring in the new commits.